components/camel-fhir/camel-fhir-component/src/main/java/org/apache/camel/component/fhir/FhirComponent.java

```diff
@@ -32,9 +32,6 @@
 @Component("fhir")
 public class FhirComponent extends AbstractApiComponent<FhirApiName, FhirConfiguration, FhirApiCollection> {
 
-    @Metadata(label = "advanced")
```

Comment:
I think removing this causes a problem with the auto generated api/docs for this. I need to check first.